portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -1,45 +1,54 @@
 package com.google.sps.servlets;
 
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Safelist;
 
 @WebServlet("/form-handler")
 public class FormHandlerServlet extends HttpServlet {
-      int contact_counter = 0;
+      int contactCounter = 0;
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      contact_counter++;
-      // Get the value entered in the form.
-      String nameValue = getParameter(request, "name-input", " ");
-      String emailValue = getParameter(request, "email-input", " ");
-      String textValue = getParameter(request, "text-input", " ");
+      contactCounter++;
+      String name = Jsoup.clean(request.getParameter("name"), Safelist.none());
+      String email = Jsoup.clean(request.getParameter("email"), Safelist.none());
+      String message = Jsoup.clean(request.getParameter("message"), Safelist.none());
+      long timestamp = System.currentTimeMillis();
```

Comment:
It's more readable to define variables close to when they're used. This definition could be moved to just above the "Entiity.newBuilder" line below, since it's not used before then.